pom.xml

```diff
@@ -966,34 +1111,75 @@
         <persistence.hibernate.hbm2ddl.auto>create</persistence.hibernate.hbm2ddl.auto>
       </properties>
     </profile>
-    
+
```

Comment:
All profiles which modify properties are placed here. This increases readability.